hooks/hookexecution/execution.go

```diff
@@ -311,3 +315,40 @@ func handleHookMutations[P any](
 
 	return payload
 }
+
+func handleModuleActivities[T any, P any](hook hooks.HookWrapper[T], activityControl privacy.ActivityControl, payload P) P {
+	// only 2 stages receive bidder request: hookstage.ProcessedAuctionRequestPayload and hookstage.BidderRequestPayload
+	// they both implement PayloadBidderRequest interface in order to execute mutations on bid request
+	var payloadData hookstage.PayloadBidderRequest
+	ok := true
+	if payloadData, ok = any(&payload).(hookstage.PayloadBidderRequest); !ok {
+		// payload doesn't have a bid request
+		return payload
+	}
+
+	privacyEnforcement := privacy.Enforcement{}
+
+	// parse hook.Module to split it to type and mame?
+	// hook.Module example: "mytest.mymodule". Can it be "rtd.mymodule" or "general.mymodule"?
```

Comment:
We need more information from the module to make this determination. Consider adding an RTD flag? .. or a Kind type of flag?

Reply:
Do you mean we need to add new property to `type HookWrapper[T any] struct`? Like `IsRTD bool`?